src/xo-app/vm/tab-disks.js

```diff
@@ -484,102 +560,7 @@ export default class TabDisks extends Component {
-                    <td> {sr &&
-                      <XoSelect
-                        onChange={sr => migrateVdi(vdi, sr)}
-                        xoType='SR'
-                        predicate={sr => sr.$pool === vm.$pool && isSrWritable(sr)}
-                        labelProp='name_label'
-                        value={sr}
-                        useLongClick
-                      >
-                        <Link to={`/srs/${sr.id}`}>{sr.name_label}</Link>
-                      </XoSelect>
-                    }
-                    </td>
-                    {vm.virtualizationMode === 'pv' && <td>
-                      <Toggle
-                        value={vbd.bootable}
-                        onChange={bootable => setBootableVbd(vbd, bootable)}
-                      />
-                    </td>}
-                    <td>
-                      <StateButton
-                        disabledLabel={_('vbdStatusDisconnected')}
-                        disabledHandler={connectVbd}
-                        disabledTooltip={_('vbdConnect')}
-
-                        enabledLabel={_('vbdStatusConnected')}
-                        enabledHandler={disconnectVbd}
-                        enabledTooltip={_('vbdDisconnect')}
-
-                        disabled={!(vbd.attached || isVmRunning(vm))}
-                        handlerParam={vbd}
-                        state={vbd.attached}
-                      />
-                    </td>
-                    <td className='text-xs-right'>
-                      <Tooltip content={_('vdiMigrate')}>
-                        <ActionRowButton
-                          icon='vdi-migrate'
-                          handler={this._migrateVdi}
-                          handlerParam={vdi}
-                        />
-                      </Tooltip>
-                      {!vbd.attached &&
-                        <span>
-                          <Tooltip content={_('vdiForget')}>
-                            <ActionRowButton
-                              icon='vdi-forget'
-                              handler={deleteVbd}
-                              handlerParam={vbd}
-                            />
-                          </Tooltip>
-                          <Tooltip content={_('vdiRemove')}>
-                            <ActionRowButton
-                              icon='vdi-remove'
-                              handler={deleteVdi}
-                              handlerParam={vdi}
-                            />
-                          </Tooltip>
-                        </span>
-                      }
-                    </td>
-                  </tr>
-                })}
-              </tbody>
-            </table>
+        <Col> {!isEmpty(vbds) ? <SortedTable collection={vbds} columns={COLUMNS} individualActions={INDIVIDUAL_ACTIONS} />
             : <h4 className='text-xs-center'>{_('vbdNoVbd')}</h4>
```

Comment:
You can remove this alternative, `SortedTable` properly handle the case with no items.